synergy: 1.13.1.41 -> 1.14.1.32

osintowl opened the pull request:
<!--
To help with the large amounts of pull requests, we would appreciate your
reviews of other pull requests, especially simple package updates. Just leave a
comment describing what you have tested in the relevant package/service.
Reviewing helps to reduce the average time-to-merge for everyone.
Thanks a lot if you do!
List of open PRs: https://github.com/NixOS/nixpkgs/pulls
Reviewing guidelines: https://nixos.org/manual/nixpkgs/unstable/#chap-reviewing-contributions
-->

###### Motivation for this change


###### Things done

<!-- Please check what applies. Note that these are not hard requirements but merely serve as information for reviewers. -->

- Built on platform(s)
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/#sec-conf-file))
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [21.11 Release Notes (or backporting 21.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2111-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

nh2 committed:
nixos/smokeping: Remove bash wrapper, refactor.

The bash wrapper process served no purpose, and systemd directly controlling the
processes is more reliable / more responsive to systemctl commands.

nh2 committed:
nixos/smokeping: Add host option.

Allows setting smokeping to not listen on the public Internet.

nh2 committed:
nixos/smokeping: Make default imgUrl relative.

This avoids a common problem:

Until now, port forwarding to multiple hosts running smokeping did not work;
they all show the data of the first smokeping instance.
That ws because the image URLs generated by smokeping are absolute
(`imgurl` setting).
Consequently, if you ran
  ssh node-1 -L 8081:localhost:8081
  ssh node-2 -L 8081:localhost:8082
  ssh node-3 -L 8081:localhost:8083
and try to open http://localhost:8081, http://localhost:8082 and
http://localhost:8083, they all would show the images of node-1!

Using a relative `imgurl` fixes that.
As per smokeping docs on `imgurl`:

> Either an absolute URL to the `imgcache` directory or one relative to the
> directory where you keep the SmokePing cgi.

nh2 committed:
nixos/smokeping: Remove partOf to ensure restarts work.

Details on https://github.com/NixOS/nixops/issues/1063#issuecomment-453253666.

`partOf` makes that if `smokeping.service` is stopped, `thttpd.service` will
be stopped as well.
(But not that `thttpd` will be started when `smokeping` is started).

Once `thttpd.service` is stopped that way, `Restart = always` will not apply.

When the smokeping config options are changed, NixOS's `switch-configuration.pl`
will stop `smokeping` (whit shuts down thttpd due to `partOf`), and then restart
smokeping; but this does not start thttpd.
As a result, thttpd will be off after changing the config, which isn't desired.

This commit fixes it by removing the `partOf`, which makes `Restart` work
as expected.

nh2 committed:
nixos/smokeping: Use requiredBy instead of wantedBy.

This makes switch-configuration fail if something is wrong with it,
which is desired especially for NixOps deployments.

evils committed:
nixos/fancontrol: restart on-failure

onny committed:
nixos/dokuwiki: add missing option enable

kfollesdal committed:
python3Packages.databricks-connect: Fix, use jdk8

erikarvstedt committed:
electrs/update.sh: ensure tag is checked out

Cloning a tag-named branch introduced a supply chain attack vector,
because branch and tag contents might differ.
Now the hashed worktree always corresponds to the tag that is GPG-verified.

manveru committed:
stage2: use atomic bind mounts

oxalica committed:
rust-analyzer: 2021-09-20 -> 2021-10-25

proux01 committed:
coqPackages.coqeal: master, add dependency

In order to include matrix normal forms in
CoqEAL (https://github.com/coq-community/coqeal/pull/54)
we add a dependency to mathcomp-real-closed.

oxalica committed:
{timedoctor,hyperspace-cli}: mark as broken

oxalica committed:
vscode-lldb: pin vsce version to fix build

superherointj committed:
ocamlPackages.piaf: init 0.1.0

superherointj committed:
ocamlPackages.gluten: init 0.2.1

superherointj committed:
ocamlPackages.gluten-lwt: init 0.2.1

superherointj committed:
ocamlPackages.gluten-lwt-unix: init 0.2.1

lovesegfault committed:
makeRustPlatform: allow to easily override stdenv

Ma27 committed:
linux: make sure that `src`/`version` actually refer to the declaring file

With this change it's ensured that `builtins.unsafeGetAttrPos` actually
points to `<nixpkgs/pkgs/os-specific/linux/kernel/linux-x.y.nix>` when
retrieving the position of `src` or `version` of
`linuxPackages.kernel`.

This is relevant to make sure that ofborg pinging maintainers on kernel updates
actually works[1]. While the underlying issue should be fixed in ofborg
or Nix itself, this is IMHO a pragmatic change to ensure that all kernel
maintainers are automatically notified on updates.

[1] https://github.com/NixOS/nixpkgs/pull/143113#issuecomment-953319957

r-ryantm committed:
python38Packages.preshed: 3.0.5 -> 3.0.6

flexagoon reviewed (changes requested):
##### Reviewed points

- [x] package name fits guidelines
- [x] package version fits guidelines
- [ ] package build on x86_64-linux
- [ ] executables tested on x86_64-linux
- [ ] all depending packages build
- [ ] fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md)

##### Comments

##### Issues
Package doesn't build and doesn't match CONTRIBUTING.md

##### How to fix
 - Please change your commit message to match the PR name (`synergy: 1.13.1.41 -> 1.14.1.32`).
 - Change the sha256 hash, as I specified in the review below
 - In the first line of `default.nix`, inside of the curly brackets, add `pkg-config`, `gdk-pixbuf` and `libnotify`, separated by commas.
 - Add `pkg-config` to nativeBuildInputs.
 - Add `gdk-pixbuf` and `libnotify` to buildInputs.

You can try to do this by yourself if you want some practice with nix, but if you feel like it's too hard for you, just tell me and I'll open my own pull request.

vbgl committed:
why3: remove spurious camlp5 dependency

cpcloud committed:
abseil-cpp: add cxxStandard input

cpcloud committed:
abseil-cpp: format

cpcloud committed:
grpc: remove explicit cxx17 requirement coming from abseil-cpp

cpcloud committed:
all-packages: pass cxxStandard to or-tools

yuyuyureka committed:
hd-idle: package rewrite

Someone ported the original hd-idle to Golang and is maintaining it.
This replaces the hd-idle package with the actively maintained Golang
port. I assume nobody will need unmaintained original version since
the Golang port can do everything the original version can do.

Commit:
python3Packages.pyclip: init at 0.5.4

oxalica committed:
plasma-workspace: fix systemd unit detection

Our units are links in systemd unit directory, which reports state `linked` or
`linked-runtime`. This fixes systemdBoot config and the option
`services.xserver.desktopManager.plasma5.runUsingSystemd` for plasma 5.23.

oxalica committed:
nixosTest/plasma5-systemd-start: init

r-burns committed:
python3Packages.xdis: 5.0.11 -> 6.0.2

r-burns committed:
python3Packages.uncompyle6: 3.7.4 -> 3.8.0

ayazhafiz committed:
wheel-filename: init at 1.3.0

ayazhafiz committed:
headerparser: init at 0.4.0

ayazhafiz committed:
entry-points-txt: init at 0.1.0

ayazhafiz committed:
wheel-inspect: init at 1.7.0

wheel-inspect has a dependency on readme_renderer~=24.0.0, but the
version of readme_renderer in nixpkgs is at major 29 (currently). To
deal with this, we build a readme_renderer version at 24 inline with
the wheel-inspect derivation.

ayazhafiz committed:
maintainers: add ayazhafiz

ayazhafiz committed:
maintainers: add ayazhafiz to entry-points-txt, headerparser, wheel-filename, wheel-inspect

polykernel committed:
lib/lists: mutuallyExclusive function optimization

The current implementation of `mutuallyExclusive` builds a new list with
length subtracted by one on every recursive call which is expensive. When
b is empty, the function still traverses a in its entirety before returning
a result.

The new implementation uses `any` to check if each element of list b is in
list a using `elem`. This maintains short circuiting when list a or b is empty
and has a worst case time complexity of O(nm).

DamienCassou committed:
Merge pull request #143612 from r-ryantm/auto-update/crowdin-cli

xaverdh committed:
linux_latest: 5.14 -> 5.15

xaverdh committed:
nixos/kernel: test all upstream kernels, reference linuxKernel directly

flokli committed:
Merge pull request #143641 from Ma27/bump-neomutt

neomutt: 20211022 -> 20211029

roberth committed:
Merge pull request #143207 from hercules-ci/evalModules-add-extendModules-and-type

lib.evalModules: Add extendModules and type to result

prusnak committed:
monado: fix build failure related to VK_NULL_HANDLE change

r-ryantm committed:
python38Packages.azure-mgmt-eventhub: 9.1.0 -> 10.0.0

fabaff committed:
Merge pull request #144018 from fabaff/bump-icmplib

python3Packages.icmplib: 3.0.1 -> 3.0.2

fabaff committed:
Merge pull request #143991 from fabaff/bump-ibis

python3Packages.ibis: 1.6.0 -> 3.2.0

SuperSandro2000 committed:
Merge pull request #143125 from kurnevsky/pidgin-ssl-fix

SuperSandro2000 committed:
Merge pull request #143675 from Synthetica9/pname-version-plexamp

fabaff committed:
Merge pull request #143975 from fabaff/bump-sniffglue

sniffglue: 0.13.1 -> 0.14.0

fabaff committed:
Merge pull request #143979 from fabaff/bump-sedutil

sedutil: 1.15.1 -> 1.20.0

Ma27 committed:
Merge pull request #144079 from xaverdh/linux-5.15

Linux 5.15

primeos committed:
nixos/tests/chromium: Add a workaround to fix the test for M96+

Some upstream changes broke the automatic fallback to SwiftShader.
Without this workaround the GPU initialization fails (apparently due to requiring Vulkan):
machine # libva error: vaGetDriverNameByIndex() failed with unknown libva error, driver_name = (null)
machine # [1001:1001:1101/104304.000625:ERROR:viz_main_impl.cc(161)] Exiting GPU process due to errors during initialization
machine # libva error: vaGetDriverNameByIndex() failed with unknown libva error, driver_name = (null)
machine # [1052:1052:1101/104305.060496:ERROR:viz_main_impl.cc(161)] Exiting GPU process due to errors during initialization
machine # [1084:1084:1101/104305.165898:ERROR:angle_platform_impl.cc(44)] Display.cpp:894 (initialize): ANGLE Display::initialize error 0: Internal Vulkan error (-3): Initialization of an object could not be completed for implementation-specific reasons, in ../../third_party/angle/src/libANGLE/renderer/vulkan/RendererVk.cpp, initialize:1048.
machine # [1084:1084:1101/104305.171191:ERROR:gl_surface_egl.cc(782)] EGL Driver message (Critical) eglInitialize: Internal Vulkan error (-3): Initialization of an object could not be completed for implementation-specific reasons, in ../../third_party/angle/src/libANGLE/renderer/vulkan/RendererVk.cpp, initialize:1048.
machine # [1084:1084:1101/104305.178707:ERROR:gl_surface_egl.cc(1382)] eglInitialize SwANGLE failed with error EGL_NOT_INITIALIZED
machine # [1084:1084:1101/104305.180111:ERROR:gl_ozone_egl.cc(20)] GLSurfaceEGL::InitializeOneOff failed.
machine # [1084:1084:1101/104305.189760:ERROR:viz_main_impl.cc(161)] Exiting GPU process due to errors during initialization
machine # [1092:1092:1101/104305.233470:ERROR:gpu_init.cc(457)] Passthrough is not supported, GL is disabled, ANGLE is

This workaround restores the previous result:
machine # [1004:1004:1101/104551.131190:ERROR:gpu_init.cc(457)] Passthrough is not supported, GL is swiftshader, ANGLE is

The workaround is only required for Chromium, not Google Chrome.

alyssais committed:
nixos/qemu-vm: use qemu_kvm

qemu_kvm is only built for one architecture, so it's smaller and takes
MUCH less time to build if it has to be built from source.  And this
module doesn't support running a VM for one architecture from another
architecture, so the one architecture is all we'll need.

expipiplus1 committed:
Merge pull request #144086 from prusnak/monado

monado: fix build failure related to VK_NULL_HANDLE change

mweinelt committed:
home-assistant: pin influxdb-client at 1.21.0

kevincox committed:
Merge pull request #144062 from IvarWithoutBones/dotnet-module/tests

buildDotnetModule: add support for running unit tests

github-actions[bot] committed:
Merge master into staging-next

r-ryantm committed:
python38Packages.django_3: 3.2.8 -> 3.2.9

primeos committed:
Merge pull request #144099 from primeos/nixos-tests-chromium

nixos/tests/chromium: Add a workaround to fix the test for M96+

mweinelt committed:
Merge pull request #143753 from r-ryantm/auto-update/python3.9-influxdb-client

gebner committed:
elan: 1.2.0 -> 1.3.0

bobby285271 committed:
Merge pull request #144096 from Atemu/update/anki-bin

anki-bin: 2.1.48 -> 2.1.49

SuperSandro2000 committed:
Merge pull request #144107 from r-ryantm/auto-update/python38Packages.django_3

bobby285271 committed:
Merge pull request #144065 from r-ryantm/auto-update/ryujinx

ryujinx: 1.0.7086 -> 1.0.7094

Ma27 committed:
Merge pull request #143234 from Ma27/ofborg-kernel-pings

 linux: make sure that src/version actually refer to the declaring file

SuperSandro2000 committed:
Merge pull request #144109 from gebner/elan130

SuperSandro2000 committed:
maintainer: sort

r-ryantm committed:
python38Packages.github3_py: 2.0.0 -> 3.0.0

erikarvstedt committed:
electrs: 0.9.1 -> 0.9.2

whonore committed:
bitwise: 0.42 -> 0.43

yuuyins committed:
maintainers: add yuu

yuuyins committed:
python3Packages.django-debug-toolbar: init at 3.2.2

r-ryantm committed:
python38Packages.google-re2: 0.2.20210901 -> 0.2.20211101

prusnak committed:
Merge pull request #144114 from erikarvstedt/electrs-0.9.2

electrs: 0.9.1 -> 0.9.2

mvnetbiz committed:
home-assistant: pin aiounifi to v28

SuperSandro2000 committed:
Merge pull request #144113 from SuperSandro2000/sort

AndersonTorres committed:
Merge pull request #143708 from AndersonTorres/new-misc

ucg: rewrite tests

bobby285271 committed:
Merge pull request #144116 from whonore/update-bitwise-0.43

bitwise: 0.42 -> 0.43

SuperSandro2000 committed:
Merge pull request #144060 from r-ryantm/auto-update/qbe

SuperSandro2000 committed:
Merge pull request #144112 from r-ryantm/auto-update/python38Packages.github3_py

bobby285271 committed:
Merge pull request #143710 from wahjava/update-flannel

cni-flannel-plugin: 1.1 -> 1.0.0

SuperSandro2000 committed:
Merge pull request #144106 from alyssais/qemu-kvm

Synthetica9 committed:
sway: inherit tests from unwrapped package

nh2 committed:
nixos/smokeping: Don't listen on all interfaces by default.

In general, NixOS services are configured such that by default
they are not exposed to the Internet for security, see #100192.

SuperSandro2000 committed:
Merge pull request #143567 from r-ryantm/auto-update/qcad

bobby285271 committed:
Merge pull request #143071 from WolfangAukang/pocket-casts-fix

pocket-casts: set electron version on top-level

nh2 committed:
nixos/rl-2111: Document smokeping service updates

cpcloud committed:
crc32c: 1.1.0 -> 1.1.2 (#143839)

bobby285271 committed:
Merge pull request #143591 from r-ryantm/auto-update/qjackctl

qjackctl: 0.9.4 -> 0.9.5

SuperSandro2000 committed:
Merge pull request #144125 from Synthetica9/sway-inherit-tests

fabaff committed:
Merge pull request #144119 from mvnetbiz/home-assistant

home-assistant: pin aiounifi to v28

SuperSandro2000 committed:
Merge pull request #144117 from r-ryantm/auto-update/python38Packages.google-re2

fabaff committed:
python3Packages.hstspreload: 2021.10.1 -> 2021.11.1

SuperSandro2000 committed:
Merge pull request #137662 from yuuyins/django-debug-toolbar

SuperSandro2000 committed:
Merge pull request #142232 from jbellerb/libressl

fabaff committed:
python3Packages.isbnlib: 3.10.8 -> 3.10.9

FabianGD committed:
python3Packages.qutip: init at 4.6.2

fabaff committed:
python3Packages.rapidfuzz: 1.8.0 -> 1.8.2

figsoda committed:
statix: 0.3.4 -> 0.3.5

markuskowa committed:
libxc: 5.1.6 -> 5.1.7

fabaff committed:
Merge pull request #144132 from fabaff/bump-isbnlib

python3Packages.isbnlib: 3.10.8 -> 3.10.9

fabaff committed:
Merge pull request #144130 from fabaff/bump-hstspreload

python3Packages.hstspreload: 2021.10.1 -> 2021.11.1

Commit:
synergy: 1.13.1.41 -> 1.14.1.32

added packages  suggested by flexagoon, reaching out to see how they got the sha256 hash before updating that.

jojosch committed:
dbeaver: 21.2.3 -> 21.2.4

nh2 committed:
nixos/smokeping: Add nh2 to maintainers

figsoda committed:
Merge pull request #144138 from figsoda/update-statix

statix: 0.3.4 -> 0.3.5

SuperSandro2000 committed:
Merge pull request #144136 from markuskowa/upd-libxc

SuperSandro2000 committed:
Merge pull request #144135 from fabaff/bump-rapidfuzz

SuperSandro2000 commented on pkgs/applications/misc/synergy/default.nix:
```diff
@@ -5,14 +5,14 @@
     rev = "${version}-stable";
     fetchSubmodules = true;
-    sha256 = "1phg0szc9g018zxs5wbys4drzq1cdhyzajfg45l6a3fmi6qdi1kw";
+    sha256 = "1hb62znagdbgs2yf9fp6mb7ixqya6lc4hjk2fhrfi07qy18dh537";
```
You are missing submodules in there.

We get those hashes by invalidating the old one and trying to build once which fails with a hash mismatch and then gives you the correct one.

5A4B48 replied:
thank you i'm sorry I'm new to this I appreciate the handholding I'm hoping to get better at this and actually be able to help without wasting ya'lls time.

talyz replied:
You're still missing the last character in the hash. You can try out your changes locally before submitting them by running `nix-build . -A synergy` from the root of the repo.

Commit:
updated sha256 hash

oxalica committed:
nodePackages: update

github-actions[bot] committed:
Merge master into staging-next

collares committed:
lean: 3.33.0 -> 3.34.0

collares committed:
lean: 3.34.0 -> 3.35.0

lovesegfault committed:
Merge pull request #144143 from lovesegfault/make-rust-platform-stdenv-redux

makeRustPlatform: allow to easily override stdenv

gebner committed:
Merge pull request #144145 from collares/lean-3.35

lean: 3.33.0 -> 3.35.0

jraygauthier committed:
nxpmicro-mfgtools: 1.4.72 -> 1.4.165

Also:

 -  Add support for provided bash completions.
 -  Add myself as maintainer.

maaslalani committed:
slides: 0.6.2 -> 0.7.2

jojosch committed:
protoc-gen-twirp_php: 0.7.5 -> 0.8.0

flexagoon committed:
sierra-breeze-enhanced: init at 1.0.3 (#143696)

SuperSandro2000 committed:
Merge pull request #140769 from 06kellyjac/opa

SuperSandro2000 committed:
Merge pull request #143647 from r-ryantm/auto-update/etcher

SuperSandro2000 committed:
Merge pull request #142814 from WolfangAukang/buttercup

jbedo committed:
Merge pull request #144149 from NixOS/r-updates

rPackages: package updates

SuperSandro2000 committed:
Merge pull request #143197 from WolfangAukang/markmind

r-ryantm committed:
python38Packages.pex: 2.1.53 -> 2.1.54

markuskowa committed:
Merge pull request #143470 from ck3d/vdrPlugins.epgsearch-2.4.1

vdrPlugins.epgsearch: 20191202 -> 2.4.1

SuperSandro2000 committed:
Merge pull request #144148 from jojosch/protoc-gen-twirp_php-0.8.0

protoc-gen-twirp_php: 0.7.5 -> 0.8.0

SuperSandro2000 committed:
Merge pull request #144147 from maaslalani/ml-bump-slides

slides: 0.6.2 -> 0.7.2

SuperSandro2000 committed:
Merge pull request #144146 from jraygauthier/jrg/nxpmicro-mfgtools-up-to-1-4-165

SuperSandro2000 committed:
Merge pull request #144059 from r-burns/xdis-uncompyle6

python3Packages.{xdis,uncompyle6}: bump to latest to support python 3.8.12

markuskowa committed:
Merge pull request #143421 from ck3d/fix-neovim-EDITOR

nixos/neovim: Respect option defaultEditor

KAction committed:
datefudge: work correctly even if GNU date is not in PATH (#94045)

* datefudge: work correctly even if GNU date is not in PATH

Examples in manual assumes advanced features from date(1) like "last
Friday", which only available in GNU coreutils version of date(1)
utility. Without this patch, most examples from datefudge(1) manual will
fail in busybox environment, which is confusing.

* Apply suggestions from code review

* Update pkgs/tools/system/datefudge/default.nix

Co-authored-by: Sandro <sandro.jaeckel@gmail.com>

WolfangAukang committed:
threema-desktop: init at 1.0.3 (#143194)

Co-authored-by: Sandro <sandro.jaeckel@gmail.com>

markuskowa committed:
Merge pull request #143279 from r-ryantm/auto-update/vdr

vdr: 2.4.7 -> 2.5.6

WolfangAukang committed:
whalebird: init at 4.4.5 (#142885)

* whalebird: init at 4.4.5

* Update pkgs/applications/misc/whalebird/default.nix

Co-authored-by: Sandro <sandro.jaeckel@gmail.com>

aanderse committed:
kodi.packages.jellyfin: 0.7.4 -> 0.7.7

aanderse committed:
Merge pull request #143706 from aanderse/zabbix

zabbix: package updates

blitz committed:
spike: move test to installCheck phase (#143832)

Co-authored-by: Sandro <sandro.jaeckel@gmail.com>

SuperSandro2000 committed:
Merge pull request #144144 from fortuneteller2k/river

SuperSandro2000 committed:
Merge pull request #144031 from CajuM/pyclip

SuperSandro2000 committed:
fcitx5-mozc: fix build

SuperSandro2000 committed:
tdesktop: fix build

fabaff committed:
Merge pull request #144151 from r-ryantm/auto-update/python38Packages.pex

python38Packages.pex: 2.1.53 -> 2.1.54

mkg20001 committed:
Merge pull request #144139 from jojosch/dbeaver-21.2.4

SuperSandro2000 committed:
Merge pull request #143457 from ck3d/pinentry-1.2.0

SuperSandro2000 committed:
Merge pull request #144152 from aanderse/kodi.packages.jellyfin

SuperSandro2000 committed:
Merge pull request #143967 from cpcloud/remove-abseil-cxx-standard-flag

abseil-cpp: remove cxx standard flag

fabianhjr committed:
scala_2_13: 2.13.6 → 2.13.7

7c6f434c committed:
Merge pull request #143160 from thblt/amc

auto-multiple-choice: init at 1.5.1

onsails committed:
tabnine: 3.6.8 -> 3.7.25

rycee committed:
bash-preexec: fix build with Bats 1.5.0

msfjarvis committed:
fclones: 0.17.0 -> 0.17.1 (#144150)

Co-authored-by: Sandro <sandro.jaeckel@gmail.com>

SuperSandro2000 committed:
python39Packages.google-cloud-error-reporting: 1.3.0 -> 1.4.0

SuperSandro2000 committed:
Merge pull request #144024 from SuperSandro2000/fix-plasma-users

SuperSandro2000 committed:
Merge pull request #142244 from r-ryantm/auto-update/python38Packages.google-cloud-appengine-logging

SuperSandro2000 committed:
zoxide: 0.7.8 -> 0.7.9

SuperSandro2000 committed:
python39Packages.google-cloud-logging: fix build

lovesegfault committed:
Merge pull request #141884 from onsails/tabnine

tabnine: support aarch64-darwin

github-actions[bot] committed:
Merge master into staging-next

happysalada committed:
dokuwiki: add release notes

r-ryantm committed:
python38Packages.jupyter_server: 1.11.1 -> 1.11.2

r-ryantm committed:
python38Packages.sqlmap: 1.5.10 -> 1.5.11

siraben committed:
Merge pull request #144064 from ayazhafiz/init/wheel-inspect

wheel-inspect: init at 1.7.0

moni-dz committed:
river: 2021-09-30 -> 2021-11-01

marsam committed:
Merge pull request #144073 from tdemin/master

shadowsocks-rust: 1.10.7 -> 1.11.2

r-ryantm committed:
python38Packages.mypy-boto3-s3: 1.19.7 -> 1.19.8

r-ryantm committed:
ryujinx: 1.0.7094 -> 1.0.7096

jonringer committed:
ksnip: fix qt wrapping

r-ryantm committed:
cargo-msrv: 0.11.1 -> 0.12.0

github-actions[bot] committed:
Merge master into staging-next

bobby285271 committed:
Merge pull request #144192 from jonringer/fix-ksnip

ksnip: fix qt wrapping

vcunat committed:
Merge #141684: branch 'staging-next'

flexagoon committed:
mullvad-vpn: disable update notifications

andersk committed:
lxd: Add upstream patch for LXC 4.0.11 compatibility

Signed-off-by: Anders Kaseorg <andersk@mit.edu>

siraben committed:
Merge pull request #144072 from polykernel/lib-lists-mutuallyexclusive-optimization

r-ryantm committed:
python38Packages.pytools: 2021.2.8 -> 2021.2.9

mweinelt committed:
home-assistant: 2021.10.6 -> 2021.10.7

mkg20001 committed:
Merge pull request #143353 from flexagoon/mullvad-update

fabaff committed:
python3Packages.aiopg: 1.3.2 -> 1.3.3

fabaff committed:
python3Packages.pontos: 21.10.2 -> 21.11.0

mkg20001 committed:
Merge pull request #136591 from polykernel/brave-add-cmdline-args

SuperSandro2000 committed:
Merge pull request #144163 from SuperSandro2000/zoxide

ymarkus committed:
bookstack: 21.08.3 -> 21.10.3

rnhmjoj committed:
Merge pull request #144210 from evils/fancontrol

nixos/fancontrol: restart on-failure

r-ryantm committed:
meteo: 0.9.9 -> 0.9.9.1

r-ryantm committed:
python38Packages.aenum: 3.1.1 -> 3.1.2

markuskowa committed:
hwloc: 2.5.0 -> 2.6.0

globin committed:
atlassian-confluence: 7.12.2 -> 7.14.1

globin committed:
atlassian-crowd: 4.2.0 -> 4.4.0

globin committed:
atlassian-jira: 8.19.0 -> 8.20.1

mweinelt committed:
Merge pull request #144205 from NixOS/home-assistant

r-ryantm committed:
MIDIVisualizer: 6.4 -> 6.5

Artturin committed:
Merge pull request #144215 from ymarkus/bookstack

bookstack: 21.08.3 -> 21.10.3

r-ryantm committed:
python38Packages.google-cloud-bigquery-logging: 1.0.0 -> 1.0.1

r-ryantm committed:
antiprism: 0.26 -> 0.29

bobby285271 committed:
Merge pull request #144217 from r-ryantm/auto-update/meteo

meteo: 0.9.9 -> 0.9.9.1

marsam committed:
Merge pull request #144199 from andersk/lxd-core-scheduling

lxd: Add upstream patch for LXC 4.0.11 compatibility

marsam committed:
Merge pull request #144178 from marsam/update-archiver

archiver: 3.5.0 -> 3.5.1

arkivm committed:
pcm: 202107 -> 202110 (#144175)

Co-authored-by: Sandro <sandro.jaeckel@gmail.com>

Artturin committed:
Merge pull request #143616 from r-ryantm/auto-update/dapper

dapper: 0.5.6 -> 0.5.7

r-ryantm committed:
python38Packages.azure-mgmt-signalr: 0.4.0 -> 1.0.0

Artturin committed:
Merge pull request #143618 from r-ryantm/auto-update/dcm2niix

dcm2niix: 1.0.20210317 -> 1.0.20211006

Artturin committed:
Merge pull request #143627 from r-ryantm/auto-update/doctl

doctl: 1.64.0 -> 1.65.0

Artturin committed:
Merge pull request #143630 from r-ryantm/auto-update/doppler

doppler: 3.31.1 -> 3.35.0

Artturin committed:
Merge pull request #143916 from r-ryantm/auto-update/influxdb

influxdb: 1.8.9 -> 1.8.10

Artturin committed:
Merge pull request #143926 from r-ryantm/auto-update/fluxcd

fluxcd: 0.20.0 -> 0.20.1

markuskowa committed:
Merge pull request #144221 from markuskowa/upd-hwloc

hwloc: 2.5.0 -> 2.6.0

Artturin committed:
Merge pull request #143936 from r-ryantm/auto-update/osu-lazer

osu-lazer: 2021.1016.0 -> 2021.1028.0

Artturin committed:
Merge pull request #144066 from r-ryantm/auto-update/sbt-extras

sbt-extras: 2021-09-24 -> 2021-10-21

Artturin committed:
Merge pull request #144067 from r-ryantm/auto-update/scli

scli: 0.6.4 -> 0.6.5

Artturin committed:
Merge pull request #144188 from r-ryantm/auto-update/ryujinx

ryujinx: 1.0.7094 -> 1.0.7096

Artturin committed:
Merge pull request #144194 from r-ryantm/auto-update/cargo-msrv

cargo-msrv: 0.11.1 -> 0.12.0

Artturin committed:
Merge pull request #144226 from r-ryantm/auto-update/antiprism

antiprism: 0.26 -> 0.29

Artturin committed:
Merge pull request #143605 from r-ryantm/auto-update/convimg

convimg: 8.3 -> 8.9

autophagy committed:
python3Packages.black: 21.9b0 -> 21.10b0 (#144213)

Co-authored-by: Phillip Cloud <417981+cpcloud@users.noreply.github.com>
Co-authored-by: Sandro <sandro.jaeckel@gmail.com>

Artturin committed:
Merge pull request #143608 from r-ryantm/auto-update/qtads

qtads: 3.1.0 -> 3.2.0

Artturin committed:
Merge pull request #143574 from r-ryantm/auto-update/cargo-udeps

cargo-udeps: 0.1.23 -> 0.1.24

Artturin committed:
Merge pull request #143575 from r-ryantm/auto-update/cariddi

Artturin committed:
Merge pull request #143599 from r-ryantm/auto-update/cloud-sql-proxy

cloud-sql-proxy: 1.25.0 -> 1.26.0

Artturin committed:
Merge pull request #143600 from r-ryantm/auto-update/cloudflared

cloudflared: 2021.9.2 -> 2021.10.5

Artturin committed:
Merge pull request #143603 from r-ryantm/auto-update/codeql

codeql: 2.6.2 -> 2.7.0

SuperSandro2000 committed:
Merge pull request #144162 from SuperSandro2000/google

Artturin committed:
Merge pull request #143613 from r-ryantm/auto-update/cytoscape

cytoscape: 3.8.2 -> 3.9.0

Artturin committed:
Merge pull request #143622 from r-ryantm/auto-update/dictu

dictu: 0.20.0 -> 0.22.0

SuperSandro2000 committed:
Merge pull request #144211 from fabaff/bump-aiopg

Artturin committed:
Merge pull request #143628 from r-ryantm/auto-update/dolt

dolt: 0.28.4 -> 0.32.1

Artturin committed:
Merge pull request #143779 from r-ryantm/auto-update/i3-gaps

i3-gaps: 4.19.1 -> 4.20

r-ryantm committed:
python38Packages.dropbox: 11.21.0 -> 11.22.0

vcunat committed:
Merge #143252: linux: include Cherryview pinctrl driver

Artturin committed:
Merge pull request #143902 from r-ryantm/auto-update/gcsfuse

gcsfuse: 0.36.0 -> 0.37.0

Artturin committed:
Merge pull request #144224 from r-ryantm/auto-update/MIDIVisualizer

r-ryantm committed:
python38Packages.env-canada: 0.5.14 -> 0.5.15

Artturin committed:
grapejuice: 3.60.14 -> 3.64.16

r-ryantm committed:
python38Packages.fastecdsa: 2.2.1 -> 2.2.2

tfc committed:
Merge pull request #142747 from helsinki-systems/fix/test-runner-execute

nixos/test-runner: Fix execute() flakiness

prusnak committed:
Merge pull request #144242 from r-ryantm/auto-update/python38Packages.fastecdsa

python38Packages.fastecdsa: 2.2.1 -> 2.2.2

ceedubs committed:
metals: remove maintainer ceedubs (me)

I no longer use Scala/Metals, so I'm not a good maintainer for this package. Luckily fabianhjr and tomahna do a great job keeping it up to date 🎉

SuperSandro2000 committed:
python39Packages.google-cloud-datastore: disable tests which require credentials

SuperSandro2000 committed:
Merge pull request #144246 from mayflower/atlassian-security-updates

bobby285271 committed:
Merge pull request #144248 from ceedubs/patch-1

metals: remove maintainer ceedubs (me)

SuperSandro2000 committed:
ib-controller: remove phases

SuperSandro2000 committed:
Merge pull request #142851 from oxalica/rust-analyzer

SuperSandro2000 committed:
Merge pull request #144251 from SuperSandro2000/ib-controller

SuperSandro2000 committed:
Merge pull request #144249 from SuperSandro2000/google-cloud-datastore

ryantm committed:
Merge pull request #143360 from r-ryantm/auto-update/typos

typos: 1.1.9 -> 1.2.0

Artturin committed:
Merge pull request #144243 from Artturin/grapejuiceupdate

Commit:
builds and runs on amd64 finally got it to build and tested it.

Commit:
Merge branch 'patch-1' of github.com:5A4B48/nixpkgs into patch-1

5A4B48 commented:
Got it to build and run on my machine thanks for the tip! You all rock! let me know what to do next really appreciate it! I'd love to start backporting it to nixos 21.05 let me know next steps, hopefully I can provide better value in the future with less handholding.

jonringer commented:
How to rebase:

```
# finding the common merge base will avoid pinging all CODEOWNERs
common=$(git merge-base origin/master origin/staging)
git rebase --onto=$common HEAD~1 # or however many commits you want to pull
git push .. ... --force
```
then change the base branch in the github PR from `master` -> `staging`

See https://nixos.org/nixpkgs/manual/#submitting-changes-staging-branch for more details on staging branch

5A4B48 commented:
Thank you and sorry!

On Tue, Nov 2, 2021 at 1:40 PM Jonathan Ringer ***@***.***>
wrote:

> How to rebase:
>
> # finding the common merge base will avoid pinging all CODEOWNERs
> common=$(git merge-base origin/master origin/staging)
> git rebase --onto=$common HEAD~1 # or however many commits you want to pull
> git push .. ... --force
>
> then change the base branch in the github PR from master -> staging
>
> See https://nixos.org/nixpkgs/manual/#submitting-changes-staging-branch
> for more details on staging branch
>
> —
> You are receiving this because you modified the open/close state.
> Reply to this email directly, view it on GitHub
> <https://github.com/NixOS/nixpkgs/pull/143683#issuecomment-958029334>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AEAHD3FDDRZJKLD5BTPS4O3UKA5DVANCNFSM5G7TDJNQ>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
>


-- 
Zachary Hunsaker
Interim Security Engineer II
NCAR - Wyoming Supercomputing Center
***@***.***
307-996-4327

##########################################
GCIH, GWAPT, GPYC, GIAC Advisory Board, CHFI
